main.tf

```diff
@@ -53,15 +56,22 @@ data "aws_eks_cluster" "this" {
   name  = var.cluster_name
 }
 
+data "aws_subnet" "private" {
```

Comment:
No reason that these would be always private subnets, correct? Can we update to a more ambiguous name?

Reply:
It's just a string... therefore changed to a more generic non-saying name "default" :D